iotdb/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -465,6 +501,15 @@ public TSExecuteStatementResp executeStatement(TSExecuteStatementReq req) throws
         return getTSExecuteStatementResp(TS_StatusCode.ERROR_STATUS, e.getMessage());
       }
 
+      try {
```

Comment:
why run setConsistencyLevel here...

Reply:
Do you have a better suggestion? If don't run here,  I need to change the logic in `IoTDBStatement`, it may be a lit bit complicated.